.github/workflows/build-wheels-defined.yml

```diff
@@ -0,0 +1,278 @@
+name: defined-wheels
+on:
+  workflow_dispatch:
+    inputs:
+      packages:
+        description: >
+          Generate wheels for given packages separated by space.
+          Requirement specifiers and markers can be used.
+          For example esptool~=4.5 esp-coredump~=1.2;sys_platform!='win32'
+        type: string
+        required: true
+
+      os_ubuntu_latest:
+        description: Build on ubuntu-latest(x86_64)
+        type: boolean
+        required: false
+        default: true
+      os_windows_latest:
+        description: Build on windows-latest(x86_64)
+        type: boolean
+        required: false
+        default: true
+      os_macos_latest:
+        description: Build on macos-latest(x86_64)
+        type: boolean
+        required: false
+        default: true
+      os_macos_arm64:
+        description: Build on macos M1(aarch64)
+        type: boolean
+        required: false
+        default: true
+      os_linux_armv7:
+        description: Build on linux armv7(aarch32)
+        type: boolean
+        required: false
+        default: true
+      os_linux_arm64:
+        description: Build on linux arm64(aarch64)
+        type: boolean
+        required: false
+        default: true
+
+jobs:
+  ubuntu-latest:
+    name: linux x86_64
+    if: ${{ inputs.os_ubuntu_latest }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.8', '3.9', '3.10', '3.11']
```

Comment:
Is it possible to have 3.12 as well?

Reply:
Yes, it should be possible. I will try to add, this.